views/tables/ChartsTable/index.jsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react'
+import { Typography } from '@mui/material'
+import moment from 'moment'
+import { Link } from 'react-router-dom'
+
+import { routes } from '../../routes/routes'
+import Table from '../Table'
+import TableMenu from './TableMenu'
+
+export const DATE_FORMAT = 'MM/DD/YYYY'
+const ChartsTable = ({
+  charts,
+  sortable,
+  onDelete,
+  onDuplicate,
+  onShare,
+  user,
+}) => {
+  const headers = [
+    {
+      dataProperty: 'title',
+      label: 'Chart',
+      sortable: !!sortable,
+    },
+    {
+      dataProperty: 'updatedAt',
+      label: 'Date Modified',
+      sortable: !!sortable,
+    },
+    {
+      dataProperty: 'owner',
+      label: 'Created By',
+      sortable: !!sortable,
+    },
+    {
+      dataProperty: 'info',
+      dataAlign: 'right',
+      label: '',
+      sortable: false,
+    },
+  ]
+  const cellRenderer = (chart, property) => {
+    switch (property) {
+      case 'title':
+        const viewChart = routes.viewChart(chart._id)
+
+        return (
+          <Link className="Link--unstyled" to={viewChart}>
+            <Typography variant="body1" sx={{ color: 'text.primary' }}>
+              {chart[property]}
+            </Typography>
+          </Link>
+        )
+      case 'updatedAt':
+        return chart[property]
+          ? moment(chart[property]).format(DATE_FORMAT)
```

Comment:
I wonder if this is the time to use a different time library and slowly phase this out

Reply:
IMO we should have a separate stream of work to wrap moment out for something else. It's not used in a ton of places so wouldn't be too much of a burden.